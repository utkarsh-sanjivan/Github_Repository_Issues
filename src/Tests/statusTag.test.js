import React from 'react';
import { create } from "react-test-renderer";
import StatusTag from '../component/StatusTag/index';
import { CheckOutlined } from '@ant-design/icons';

test('Test Status Tag Component', () => {
  const component = create(
    <StatusTag
        color='#fff'
        icon={<CheckOutlined/>}
        text={'Test Text'}
    />
);
  expect(component.toJSON()).toMatchSnapshot()
});