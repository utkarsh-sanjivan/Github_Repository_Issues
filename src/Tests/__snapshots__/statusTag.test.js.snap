// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Status Tag Component 1`] = `
<span
  className="ant-tag ant-tag-has-color issue-status-tag"
  style={
    Object {
      "backgroundColor": "#fff",
    }
  }
  text="Test Text"
>
  <span
    aria-label="check"
    className="anticon anticon-check"
    role="img"
  >
    <svg
      aria-hidden="true"
      className=""
      data-icon="check"
      fill="currentColor"
      focusable="false"
      height="1em"
      viewBox="64 64 896 896"
      width="1em"
    >
      <path
        d="M912 190h-69.9c-9.8 0-19.1 4.5-25.1 12.2L404.7 724.5 207 474a32 32 0 00-25.1-12.2H112c-6.7 0-10.4 7.7-6.3 12.9l273.9 347c12.8 16.2 37.4 16.2 50.3 0l488.4-618.9c4.1-5.1.4-12.8-6.3-12.8z"
      />
    </svg>
  </span>
  <span>
    Test Text
  </span>
</span>
`;
